keyboards/helix/rules.mk

```diff
@@ -46,30 +42,30 @@ F_USB = $(F_CPU)
 #   atmega32a    bootloadHID
```

Comment:
```suggestion
#   ATmega32A    bootloadHID
#   ATmega328P   USBasp
```

And remove the `F_CPU`, `ARCH`, `F_USB` and `INTERRUPT_CONTROL_ENDPOINT` blocks.

Reply:
Thanks.